sdks/python/apache_beam/runners/common.py

```diff
@@ -98,41 +98,6 @@ def logging_name(self):
     return self.step_name
 
 
-# TODO(BEAM-4028): Move DataflowNameContext to Dataflow internal code.
```

Comment:
This isn't referenced in Beam code, but it looks like the definition still hasn't been moved to Dataflow internal code, and we reference it there.

Reply:
I'm sending a change to remove the internal reference. Once that's done I'll run internal tests on this PR before submitting. Will resolve this comment when that's done.